chapters/en/slides/module8/module8_03.md

```diff
@@ -0,0 +1,367 @@
+
+```{math}
+:label: eq_label
+m_{j} \approx 1.9 \times 10^{27} \: \text{kg}
+```
+
+Equation {eq}`eq_label` is an example of a named equation.
+~~~
+
+Notes:
+
+If you have created an equation with a label, you can link to it from within your text and across pages.
+
+You can refer to the equation using the label that you’ve provided by using `` {eq}`eq_label` ``, just like ``{numref}`jupiter-figure` `` that we used with images.
+
+<br>
+
+Here is how we add equations to `mymarkdownfile.md`:
+
+---
+
+# Cross-referencing equations
+
+Once you add these equations to `mymarkdownfile.md` and rebuild your Jupyter Book, your book will look like:
+
+<img style="border:1px solid black;" src="/module8/equation.png" width="750"></img>
+
+Notes:
+
+Once you add these equations to `mymarkdownfile.md` and rebuild your Jupyter Book, your book will look like:
+
+<img style="border:1px solid black;" src="/module8/equation.png" width="750"></img>
+
+---
+
+# Margin content
+
+Sometimes it is helpful to add notes to the margin of a page. This is similar to _footnotes_ in a conventional book.
+
+In Jupyter Book, you can do that using `{margin}` blocks in a markdown file:
+
+~~~
+```{margin} Did you know?
+Jupiter is 11.0x larger than Earth!
+```
+~~~
+
+<br>
+
+Here is how it appears in our Jupyter Book:
+
+<img style="border:1px solid black;" src="/module8/margin.png" width="800"></img>
+
+Notes:
+
+Sometimes it is helpful to add notes to the margin of a page. This is similar to _footnotes_ in a conventional book.
+
+In Jupyter Book, you can do that using `{margin}` blocks in a markdown file:
+
+~~~
+```{margin} Did you know?
+Jupiter is 11.0x larger than Earth!
+```
+~~~
+
+<br>
+
+Here is how it appears in our Jupyter Book:
+
+<img style="border:1px solid black;" src="/module8/margin.png" width="800"></img>
+
+---
+
+# Admonitions
+
+Admonitions in Jupyter Book are visual elements that you can use to draw attention to important information.
+
+There are different types of admonitions that you can use in Jupyter Book, including _Note_, _Tip_, _Warning_, _Danger_ and _Hint_.
+
+You can define an admonition by inserting an admonition code block:
+
+~~~
+```{note}
+I'm a note!
+```
+~~~
+
+`{note}` can be replaced with `{warning}`, `{tip}`, `{danger}` and so on. This is how these admonitions will show up:
+
+<img src="/module8/admonition.png" width="600"></img>
+
+Notes:
+
+Admonitions in Jupyter Book are visual elements that you can use to draw attention to important information.
+
+There are different types of admonitions that you can use in Jupyter Book, including _Note_, _Tip_, _Warning_, _Danger_ and _Hint_.
+
+You can define an admonition by inserting an admonition code block:
+
+~~~
+```{note}
+I'm a note!
+```
+~~~
+
+`{note}` can be replaced with `{warning}`, `{tip}`, `{danger}` and so on. This is how these admonitions will show up:
+
+<img src="/module8/admonition.png" width="600"></img>
+
+---
+
+# Let's see what you've learned!
```

Comment:
I think it would be great to have a section or a few slides for code and code output. E.g. show how you can show and hide code. Maybe show that jupyter book works with interactive altair plots which they have learned about before. And show that you can [use gluing to reference a variable in the notebook](https://jupyterbook.org/en/stable/content/executable/output-insert.html). This glueing actually also works for charts which currently seems to be the easiest way to create figure captions for code output, so I think that can be worth including (my second suggestion here https://github.com/executablebooks/jupyter-book/issues/1191#issuecomment-1133026263).

Reply:
Thank you for the great suggestions. I've added slides to cover all the functionalities that you've mentioned.